docs/styleguide.md

```diff
@@ -0,0 +1,24 @@
+
+List of programming style rules
+===============================
+
+(will be re-structured as it grows)
+
+- qualified imports are given the name of the module (without path),
+  not an abbreviation (`import qualified Data.Map as Map`).
+
```

Comment:
This would also prevent us from doing things like

    import qualified Data.Aeson as Aeson
    import qualified Data.Aeson.Encode.Pretty as Aeson
    import qualified Data.Aeson.Types as Aeson

(your code, not mine)